src/iidm/converter/xml/VoltageLevelXml.cpp

```diff
@@ -91,8 +93,10 @@ void VoltageLevelXml::readCalculatedBus(VoltageLevel &voltageLevel, NetworkXmlRe
 }
 
 void VoltageLevelXml::readNodeBreakerTopology(VoltageLevel& voltageLevel, NetworkXmlReaderContext& context) const {
-    const auto& nodeCount = context.getReader().getAttributeValue<unsigned long>(NODE_COUNT);
-    voltageLevel.getNodeBreakerView().setNodeCount(nodeCount);
+    IidmXmlUtil::runUntilMaximumVersion(IidmXmlVersion::V1_1(), context.getVersion(), []() {
+        logging::Logger& logger = logging::LoggerFactory::getLogger<VoltageLevelXml>();
+        logger.info(stdcxx::format("attribute %1%.nodeCount is ignored.", NODE_BREAKER_TOPOLOGY));
+    });
```

Comment:
This log has been removed in https://github.com/powsybl/powsybl-core/pull/1299 because too noisy

Reply:
Ok removed also